raiden-cli/src/raiden.ts

```diff
@@ -0,0 +1,127 @@
+import { Raiden, RaidenChannel, RaidenChannels, ChannelState } from 'raiden-ts';
+import { LocalStorage } from 'node-localstorage';
+import { flatMap, values, filter } from 'lodash';
+import { Subscription } from 'rxjs';
+
+const DEFAULT_CONFIG = {
+  matrixServer: 'https://raidentransport.test001.env.raiden.network',
+  pfs: 'https://pfs.raidentransport.test001.env.raiden.network',
+  pfsSafetyMargin: 1.1,
+  caps: { noDelivery: true, webRTC: true },
+};
+
+export default class RaidenService {
+  private static _instance: RaidenService;
+  private static _status = 'unavailable';
+  private _raiden: Raiden;
+  private _channels: RaidenChannels;
+  private _subscriptions: Subscription[];
+
+  constructor(raiden: Raiden) {
+    this._raiden = raiden;
+    this._channels = {};
+    this._subscriptions = [];
+  }
+
+  static getInstance(): RaidenService {
```

Comment:
Again, related to the 'global instances or contextual ones', these static methods also wouldn't allow for multiple instances running on same Node process, so with the suggestion get rid of services, also try to use some local instance or Raiden instead of static methods/properties.

Reply:
I like the idea to have the possibility for multiple instances. Lets see in the above discussion how we resolve that.

Reply:
Discussed above. Thanks